include/deal.II/fe/fe_series.h

```diff
@@ -213,6 +213,12 @@ namespace FESeries
               const unsigned int            cell_active_fe_index,
               Table<dim, CoefficientType> & legendre_coefficients);
 
+    /**
+     * Return number of coefficients in each direction.
```

Comment:
I have never looked at this class -- is it clear from the context what the "directions" are, or would it be useful to describe what that means?

Reply:
direction == dimension. To me it is, but I am also the one who added the class, so I am certainly  biased ;-)